[2024-06-25T14:14:42.322+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-25T14:14:42.341+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nasa_to_s3.transform_nasa_data manual__2024-06-25T14:14:38.418493+00:00 [queued]>
[2024-06-25T14:14:42.347+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nasa_to_s3.transform_nasa_data manual__2024-06-25T14:14:38.418493+00:00 [queued]>
[2024-06-25T14:14:42.347+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 4
[2024-06-25T14:14:42.358+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_nasa_data> on 2024-06-25 14:14:38.418493+00:00
[2024-06-25T14:14:42.364+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=421) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-25T14:14:42.365+0000] {standard_task_runner.py:63} INFO - Started process 423 to run task
[2024-06-25T14:14:42.366+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nasa_to_s3', 'transform_nasa_data', 'manual__2024-06-25T14:14:38.418493+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/extractAPIdata.py', '--cfg-path', '/tmp/tmpnxev7biv']
[2024-06-25T14:14:42.368+0000] {standard_task_runner.py:91} INFO - Job 55: Subtask transform_nasa_data
[2024-06-25T14:14:42.401+0000] {task_command.py:426} INFO - Running <TaskInstance: nasa_to_s3.transform_nasa_data manual__2024-06-25T14:14:38.418493+00:00 [running]> on host 4cda6158f7cb
[2024-06-25T14:14:42.471+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test' AIRFLOW_CTX_DAG_ID='nasa_to_s3' AIRFLOW_CTX_TASK_ID='transform_nasa_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-25T14:14:38.418493+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-25T14:14:38.418493+00:00'
[2024-06-25T14:14:42.472+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-25T14:14:42.516+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-25T14:14:42.516+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-25T14:14:42.524+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=nasa_to_s3, task_id=transform_nasa_data, run_id=manual__2024-06-25T14:14:38.418493+00:00, execution_date=20240625T141438, start_date=20240625T141442, end_date=20240625T141442
[2024-06-25T14:14:42.539+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-25T14:14:42.554+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-25T14:14:42.555+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
